package.json

```diff
@@ -50,7 +50,6 @@
     "redux-thunk": "^2.3.0",
     "stream-browserify": "^3.0.0",
     "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.2",
-    "taxi-protobuf": "vulpemventures/taxi-protobuf",
```

Comment:
looks like we could remove google-protobuf too?

Reply:
Commit 119c83a (just now) removes google-protobuf